### ISSUE TITLE
Server API version not detected if UTF-8 BOM is present

### ISSUE THREAD

**Issue by carlreinke** [ultrasonic/ultrasonic#196]
The version detection interceptor is not able to extract the API version from the JSON document if the text begins with the UTF-8 BOM.  The rest of the application seems to be able handle the UTF-8 BOM.

**Comment by ogarcia** [comment]
@Tapchicoma maybe related with #174

**Comment by Tapchicoma** [comment]
@carlreinke can you provide example json with UTF-8 BOM?

**Comment by carlreinke** [comment]
[ping.view.txt](https://github.com/ultrasonic/ultrasonic/files/1998914/ping.view.txt)

**Comment by Tapchicoma** [comment]
Interesting that this caused by Jackson parser:
```java
com.fasterxml.jackson.core.JsonParseException: Unexpected character ('﻿' (code 65279 / 0xfeff)): expected a valid value (number, String, array, object, 'true', 'false' or 'null')
```

**Comment by Tapchicoma** [comment]
`VersionInterceptor` can cleanup the peeked response from this BOM characters, but let see what will be answered in Jackson issue:
https://github.com/FasterXML/jackson-core/issues/458

**Comment by Tapchicoma** [comment]
Fix will be available in next release (mostly `2.5.0`)
